docs/flexbox.md

```diff
@@ -1197,6 +1197,249 @@ const styles = StyleSheet.create({
+  },
+});
+
+export default RowGap;
+```
+
+- [`columnGap`](layout-props#columnGap) property sets the size of the gap (gutter) between an element's columns.
+
+```SnackPlayer name=Column%20Gap
+import React, { useState } from "react";
+import { View, TouchableOpacity, Text, StyleSheet, TextInput } from "react-native";
+
+const ColumnGap = () => {
+  const [columnGap, setColumnGap] = useState(10);
+
+  return (
+    <PreviewLayout
+      label="columnGap"
+      handleColumnGapChange={setColumnGap}
+      columnGap={columnGap}>
+      <View
+        style={[styles.box, { backgroundColor: "orangered" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "orange" }]}
+      />
+    </PreviewLayout>
+  );
+};
+
+const PreviewLayout = ({
+  label,
+  children,
+  values,
+  selectedValue,
+  setSelectedValue,
+  handleColumnGapChange,
+  columnGap
+}) => (
+  <View style={{ padding: 10, flex: 1 }}>
+    <Text style={styles.label}>{label}</Text>
+    <View style={{rowGap: 4}}>
+      <Text>Edit Column Gap</Text>
+      <TextInput style={styles.input} value={columnGap} onChangeText={(v) => handleColumnGapChange(Number(v)) } />
+    </View>
+    <View
+      style={[
+        styles.container,
+        {flexDirection:"row", columnGap },
+      ]}
+    >
+      {children}
+    </View>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  input: {
+    borderBottomWidth: 1,
+    paddingVertical: 3,
+    width: 50,
+    textAlign: "center",
+  },
+  container: {
+    flex: 1,
+    marginTop: 8,
+    backgroundColor: "aliceblue",
+    maxHeight: 400,
+  },
+  box: {
+    width: 50,
+    height: 80,
+  },
+  label: {
+    textAlign: "center",
+    marginBottom: 10,
+    fontSize: 24,
+  },
+});
+
+export default ColumnGap;
+```
+
+- [`gap`](layout-props#gap) property sets the size of the gap (gutter) between rows and columns. It is a shorthand for `rowGap` and `columnGap`. You can use `flexWrap` and `alignContent` alongwith `gap` to add consistent space between items.
+
+```SnackPlayer name=Gap
+import React, { useState } from "react";
+import { View, TouchableOpacity, Text, StyleSheet, TextInput } from "react-native";
+
+const Gap = () => {
+  const [gap, setGap] = useState(10);
+
+  return (
+    <PreviewLayout
+      label="gap"
+      handleGapChange={setGap}
+      gap={gap}>
+      <View
+        style={[styles.box, { backgroundColor: "orangered" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "orange" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "mediumseagreen" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "deepskyblue" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "mediumturquoise" }]}
+      />
+
+    </PreviewLayout>
+  );
+};
+
+const PreviewLayout = ({
+  label,
+  children,
+  values,
+  selectedValue,
+  setSelectedValue,
+  handleGapChange,
+  gap
+}) => (
+  <View style={{ padding: 10, flex: 1 }}>
```

Comment:
Inline styles will cause an ESLint error  when using the default ESLint config for React Native. For anything that can be in a stylesheet (is not dynamic), I think it would be better to teach the best practice.